src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1108,6 +1108,12 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			this._refreshSelectionContextKey();
 		}));
 
+		this._register(dom.addDisposableListener(xterm.raw.textarea, 'focusout', () => {
```

Comment:
Shall we make a private `_focusIn` and `_focusOut` methods so the latter share impl?

Reply:
I wondered about that... wasn't sure if it was overkill since it's just 3 lines.

Reply:
maybe could combine them and have focus(boolean)

Reply:
if this fix doesn't work, we can make `setFocus` public and when the editor is focused, `setFocus(false)` on the active instance

Reply:
Something like that yeah, but only when the new active editor doesn't have a terminal URI as you reminded me of before.